source/nutrition/nutrition_facts.cpp

```diff
@@ -58,177 +58,138 @@ namespace nutrition {
         return c;
     }
 
-    nutrition_facts nutrition_facts::operator*(const nutrition_facts& other) const {
-        nutrition_facts c(*this);
-        c *= other;
-        return c;
-    }
-
-    nutrition_facts nutrition_facts::operator/(const nutrition_facts& other) const {
-        nutrition_facts c(*this);
-        c /= other;
-        return c;
-    }
-
     nutrition_facts& nutrition_facts::operator+=(const nutrition_facts& other) {
         // General
         calories += other.calories;
+        assert(this->invariants());
```

Comment:
Fazer `assert(this->invariants());` apenas no fim da chamada de cada função mutável (sem `const`). Esse assert é só para conferir se o código está correto, mas o esperado é que ela sempre dê falso.